content/projects/kotlin/project-3/create-a-fragment/_index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Create a fragment.
+ready: True
+available_options:
+- kotlin
+content_type: project
+submission_type: repo
+prerequisites:
+  hard:
+    [
+      "topics/kotlin/delegation",
+      "topics/kotlin/delegated-properties",
+      "topics/kotlin/generics",
+      "topics/kotlin/nested-classes"
+    ]
+  soft:
+    []
+---
+
+## App overview
+In the three projects that make up this lesson, you work on an app called AndroidTrivia. The completed app is a game in which the user answers three trivia questions about Android coding. If the user answers all three questions correctly, they win the game and can share their results.
+
+![](6ae358b2f280b295.png)
+
+he AndroidTrivia app illustrates navigation patterns and controls. The app has several components:
+
+- In the title screen, shown on the left in the screenshot above, the user starts the game.
+- In the game screen with questions, shown in the middle above, the user plays the game and submits their answers.
+- The navigation drawer, shown on the right above, slides out from the side of the app and contains a menu with a header. The drawer icon  opens the navigation drawer. The navigation-drawer menu contains a link to the About page and a link to the rules of the game.
+
+The top of the app displays a colored view called the app bar, which is also known as the action bar.
+
+### Task: Explore the starter app project
+
+In this project, you work from a starter app that provides template code and fragment classes that you need as you complete the Trivia app.
+
+1 - Download the [AndroidTrivia-Starter](AndroidTrivia-Starter) Android Studio project.
```

Comment:
Should we not have this project somewhere (like a shared drive) so that can download it? I dont think it will be easy to do like this